skynet.yaml

```diff
@@ -0,0 +1,22 @@
+# Testing with Skynet - https://github.com/Workiva/skynet/tree/master/docs
+name: xlsx-unit-tests
```

Comment:
looks like this still has some xlsx copy pasta

Reply:
🤦‍♂️ good catch! I'll change that.